setup.py

```diff
@@ -43,6 +43,6 @@
     keywords='ITK InsightToolkit',
     url=r'https://itk.org/',
     install_requires=[
-        r'itk>=5.3.0'
+        r'itk>=5.4.0'
```

Comment:
5.4rc2

Reply:
We might need RC3 here, due to need for https://github.com/InsightSoftwareConsortium/ITK/pull/4444 and https://github.com/InsightSoftwareConsortium/ITKPythonPackage/issues/263.